SPECS/tpm2-tss/tpm2-tss.signatures.json

```diff
@@ -1,6 +1,5 @@
 {
  "Signatures": {
-  "tpm2-tss-4.0.1.tar.gz": "532a70133910b6bd842289915b3f9423c0205c0ea009d65294ca18a74087c950",
-  "tpm2-tss.sysusers": "14c0854c73c51bbcd6764fdafa29401f8292f5eaa8abac23899209a63fe2ed4c"
```

Comment:
Double checking if this removal is a typo ?

Reply:
We removed this from spec file in commit 1b4aadf28b71fe933c122bc6c6e58b31a42b022d.
Not sure why did not reflect in signature Json file.